test/stInitCodeTestFiller.json

```diff
@@ -8,6 +8,22 @@
             "currentTimestamp" : 1,
             "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
         },
+	"expect" : {
+            "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                "balance" : "32599",
+                "code" : "0x",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+	    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "67401",
+                "code" : "0x",
+                "nonce" : "1",
+                "storage" : {
+                }
+            }
+	},
```

Comment:
I want to do something like 
"a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
                "nonce" : "1",
            }
Because when ther is a protocol change, I will only check one value, like the nonce here. Or a single value in storage. Not the complete account, that is not that useful. Because we already have that in the post after creating the test. (or in the "old" tests)